app/assets/javascripts/discourse/app/widgets/widget.js

```diff
@@ -66,6 +66,10 @@ export function resetDecorators() {
   Object.keys(_decorators).forEach((key) => delete _decorators[key]);
 }
 
+export function resetRegistry() {
```

Comment:
Should this be called by `afterEach()` in qunit-helpers.js? 😃

Reply:
oops yes I forgot part of my stash